app/experimenter/nimbus-ui/src/components/FormOverview/index.tsx

```diff
@@ -123,34 +127,37 @@ const FormOverview = ({
         <Form.Control
           {...nameValidated("hypothesis")}
           as="textarea"
-          rows={3}
-          defaultValue={experiment?.hypothesis || ""}
+          rows={5}
+          defaultValue={experiment?.hypothesis || hypothesisDefault || ""}
         />
         <Form.Text className="text-muted">
           You can add any supporting documents here.
         </Form.Text>
         <FormErrors name="hypothesis" />
       </Form.Group>
 
-      <Form.Group controlId="application">
-        <Form.Label>Application</Form.Label>
-        <Form.Control
-          {...nameValidated("application")}
-          as="select"
-          defaultValue={experiment?.application || ""}
-        >
-          <option value="">Select...</option>
-          {application!.map((app, idx) => (
-            <option key={`application-${idx}`} value={app!.value as string}>
-              {app!.label}
-            </option>
-          ))}
-        </Form.Control>
-        <Form.Text className="text-muted">
-          Experiments can only target one Application at a time.
-        </Form.Text>
-        <FormErrors name="application" />
-      </Form.Group>
+      {experiment ? (
+        <React.Fragment>
+          <p>Application</p>
+          <p>{experiment.application}</p>
```

Comment:
You mentioned this could be a followup; if you wanted to throw it in here I think it's as simple as `{application!.find((a) => a?.value === experiment.application)?.label}`

Reply:
🔥 🔥 🔥